markdown/events/2021/bytesize-10-institutional-profiles.md

```diff
@@ -41,3 +41,250 @@ You can see the slides on HackMD: <https://hackmd.io/@jfy133/Hyc_WvXU_> (shown b
 <div class="embed-responsive embed-responsive-16by9 border shadow">
   <iframe class="embed-responsive-item" src="https://hackmd.io/@jfy133/Hyc_WvXU_" allowfullscreen></iframe>
 </div>
+
+<details markdown="1"><summary>Video transcription</summary>
+**Note: The content has been edited to make it reader-friendly**
+
+[0:54](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=54) Thank you very much. As mentioned, the overview of today’s talk is that we will be recapping the benefits of using nf-core institutional profiles, and I will describe the information you should generally gather before writing one.
+
+[1:14](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=74) Then I’ll go step-by-step through how to build such a profile, and finally how to test it.
+
+[1:20](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=80) So today, I’ll be talking about global institutional profiles, which is what I will be defining during the talk. So institutional profiles are Nextflow configuration files that are used by all nf-core pipelines to work efficiently on institutional level clusters and that are stored on the `nf-core/configs` Github repository.
+
+[1:40](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=100) There’s another thing called pipeline institutional profiles, but that is something for a future bytesize talk.
+
+[1:47](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=107) So firstly, why would you want to use an institutional profile?
+
+So these give you a lot of efficiency regarding Nextflow runs in terms of computing resources and time.
+
+So making sure that you use only what you need and not exceed it and block other people in your given cluster.
+
+[2:04](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=124) They’re very portable and contribute towards reproducibility, so that means other people can run the same command you ran, but configured to their own cluster.
+
+[2:15](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=135) So ultimately writing this `config file` saves a lot of time. You write it once and then everyone on that cluster can benefit from it.
+
+[2:25](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=145) So here’s a brief recap of what was covered in [bytesize#2](https://nf-co.re/events/2021/bytesize-2-configs).
+
+A network configuration file is a simple text file containing a set of properties, so parameters and some stuff that basically Nextflow can read and understand. This can be pipeline parameters, information on locations of files, and so on.
```

Comment:
```suggestion
A Nextflow configuration file is a simple text file containing a set of properties, so parameters and some stuff that basically Nextflow can read and understand. This can be pipeline parameters, information on locations of files, and so on.
```